lib/install/management/network.go

```diff
@@ -95,26 +99,47 @@ func (d *Dispatcher) removeNetwork(conf *config.VirtualContainerHostConfigSpec)
 		return nil
 	}
 
-	name := conf.Name
-	if network, err := d.session.Finder.Network(d.ctx, name); err != nil || network == nil {
-		log.Infof("Didn't find network %q", name)
-		log.Debugf("Didn't find network for %s", err)
+	log.Debugf("Remove bridge network based on %s", conf.ExecutorConfig.Networks["bridge"].Network.ID)
+
+	name := conf.ExecutorConfig.Networks["bridge"].Network.ID
+	moref := new(types.ManagedObjectReference)
```

Comment:
I would move it below if, since there is a chance of error and creating an additional object in this case doesn't make sense. It doesn't really affect anything, but I would like all code to do only what's necessary at the specific state.